Configurations/README

```diff
@@ -67,28 +72,40 @@ In each table entry, the following keys are significant:
                            also used when linking a program where at
                            least one of the object file is made from
                            C++ source.
-        cflags          => Flags that are used at all times when
-                           compiling C object files.
-        cxxflags        => Flags that are used at all times when
-                           compiling C++ object files.  If unset, it
-                           gets the same value as cflags.
+        cflags          => Defaults flags used when compiling C object
+                           files [4].
+        cxxflags        => Default flags used when compiling C++
+                           object files [4].  If unset, it gets the
+                           same value as cflags.
         shared_cflag    => Extra compilation flags used when
                            compiling for shared libraries, typically
                            something like "-fPIC".
+        module_cflags   => Extra compilation flags used when
+                           compiling for DSOs, typically something
+                           like "-fPIC".
 
         (linking is a complex thing, see [3] below)
         ld              => Linker command, usually not defined
                            (meaning the compiler command is used
                            instead).
                            (NOTE: this is here for future use, it's
                            not implemented yet)
-        lflags          => Flags that are used when linking apps.
-        shared_ldflag   => Flags that are used when linking shared
-                           or dynamic libraries.
-        plib_lflags     => Extra linking flags to appear just before
-                           the libraries on the command line.
+        lflags          => Default flags used when linking apps,
+                           shared libraries or DSOs [4].
```

Comment:
Why not just "Default linker flags [used each time linker is invoked]"? Side note...

Reply:
Good point.  I changed a few more accordingly.